tests/VMelnalksnis.PaperlessDotNet.Tests.Integration/Documents/DocumentClientTests.cs

```diff
@@ -94,6 +95,21 @@ public async Task Create()
 
 		updatedDocument.Title.Should().Be(update.Title);
 
+		var documentDownload = await Client.Documents.Download(id, original: true);
+
+		documentDownload.ContentType.Should().Be("text/plain");
+		documentDownload.FileName.Should().Contain("Lorem Ipsum");
+
+		using (var stream = documentDownload.Content)
+		{
+			Assert.IsNotNull(stream);
+
+			StreamReader reader = new StreamReader(stream);
```

Comment:
Should use `var` instead of specifying the type twice.
```suggestion
			using var reader = new StreamReader(stream);
```